.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     "rules": {
         "indent": [ "off", 2 ],
         "quotes": [ "warn", "double" ],
-        "semi": [ "warn", "always" ]
+        "semi": [ "warn", "always" ],
+		"no-unused-vars": [ "warn", { "args": "none" } ]
```

Comment:
This stops eslint from complaining about things like:
```js
function func(x, y) { return x; }
```
Normally it would say `y` is unused, this change prevents that error.